app/views/external_users/claims/new.html.haml

```diff
@@ -1,3 +1,10 @@
-= render partial: 'layouts/header', locals: {page_heading: t('.heading')}
+%header.main-header{role: 'banner'}
+
+  - if @claim.agfs?
+    - page_heading = t('.agfs_heading')
+  - elsif @claim.lgfs?
+    - page_heading = t('.lgfs_heading')
+
```

Comment:
Couldn't the above be put into a helper method like https://github.com/ministryofjustice/advocate-defence-payments/blob/develop/app/helpers/application_helper.rb#L101

Just so we can remove this type of logic in the view